Library/Homebrew/cmd/update-bash.sh

```diff
@@ -13,11 +14,11 @@ brew() {
 which_git() {
   local which_git
   which_git="$(which git 2>/dev/null)"
-  if [ -n "$which_git" ] && [ "/usr/bin/git" = "$which_git" ]
+  if [[ -n "$which_git" && "/usr/bin/git" = "$which_git" ]]
   then
     local active_developer_dir
```

Comment:
Just a suggestion: Keep the declaration of all local variables at the top of the function. IMHO, this keeps it more readable and it's instantly clear what is local and what is not.